consensus/metrics.go

```diff
@@ -25,6 +29,9 @@ type Metrics struct {
 	// Number of rounds.
 	Rounds metrics.Gauge
 
+	// Histogram of round duration.
+	RoundDuration metrics.Histogram
```

Comment:
I'm not sure how useful round duration is in regards to measuring the impact of database compaction

Reply:
Removed.